### DISCUSSION TITLE
Considering deploying in kubernetes

### DISCUSSION THREAD

**Discussion by alexgeek** [twentyhq/twenty#6421] [Engineering vision]
Hi all,

I'm considering deploying this into Kubernetes, is there anything I should be aware of before trying?
I don't see any helm charts or anything similar in the repo. Do we know if the app can work in HA environment? I.e. Can I have multiple twenty pods for load balancing and redundancy or will that not work?

Thanks

**Comment by FelixMalfait** [comment]
@LumosViridi kindly did a contribution for this and has an open PR to update it. https://github.com/twentyhq/twenty/pull/6356

There's no "official support" for this in the near term but hopefully one day when we get more bandwidth the core team would be able to cover it!
